ConcernsCaseWork/ConcernsCaseWork.API.Tests/Integration/CloseDecisionIntegrationTests.cs

```diff
@@ -57,9 +62,14 @@ public async Task When_Patch_Returns_200Response()
 		response.Data.DecisionId.Should().Be(decisionId);
 		response.Data.CaseUrn.Should().Be(cCase.Urn);
 
-		var dbDecision = GetContext().Decisions.Single(d => d.DecisionId == decisionId);
-		dbDecision.ClosedAt.Should().BeCloseTo(DateTimeOffset.Now, TimeSpan.FromMinutes(1));
+		var _context = GetContext();
+
+		var dbDecision = _context.Decisions.Single(d => d.DecisionId == decisionId);
+		var dbCase = _context.ConcernsCase.Single(c => c.Urn == cCase.Urn);
+		dbDecision.ClosedAt.Should().BeCloseTo(now, TimeSpan.FromMinutes(1));
 		dbDecision.SupportingNotes.Should().Be(request.SupportingNotes);
+
+		dbCase.CaseLastUpdatedAt.Should().BeCloseTo(now, TimeSpan.FromMinutes(1));
```

Comment:
We should expect the CaseLastUpdatedAt to match the UpdatedAt field from the decision. Can specifically check for that please?